Yaml files split + DB health checks

Corgam opened the pull request:
- Yaml files are now split into several folders: `common`, `development`, and `production`, with files in the `common` folder loading always and the other folder loading in specific environments
- Added DBs (SQL DB and MongoDB) health checks, so other containers dependent on it wait until the DBs are set up
- Extended the shape importer to support local files
- Split one house footprints into several chunks for easier digestion into the DB in development environments (added simple python script for that in the `scripts` folder of data pipeline)
- Improved logging of the data pipeline
- Fixed volumes (and MongoDB) not working properly in the ubuntu environments. The data will not have to be downloaded every time anymore :) )

Superschnizel commented on backend/src/BIE.DataPipeline/Metadata/MetadataDbHelper.cs:
```diff
@@ -27,13 +27,13 @@ public bool CreateConnection()
         try
         {
-            var client = new MongoClient($"mongodb://datapipeline:datapipeline@{mMetaDataDbUrl}/bci-metadata");
+            var client = new MongoClient("mongodb://datapipeline:datapipeline@metadata-database:27017/bci-metadata");
```
this will mean it will not be executable locally. Why change this?

Superschnizel commented on backend/src/BIE.DataPipeline/Program.cs:
```diff
@@ -94,34 +92,36 @@
         notEof = importer.ReadLine(out line);
         count++;
-        Console.Write($"\rLines: {count}");
+        if (count % 10000 == 0)
```
i wonder if this will improve the performance, or if it would be best to leave out the writes anyways.

Corgam replied:
Yes, I did it just to improve the logging, because before it was impossible to see anything with the spam of "Lines X". The print here is just to inform us that the pipeline is still working, but we could delete it anyways if not needed.

Corgam replied:
EDIT: Deleted the print.

Superschnizel reviewed (commented):
I only see one issue with the removal of the Environment variable as Metadata server

Corgam commented:
> I only see one issue with the removal of the Environment variable as Metadata server

Reverted my changes, and fixed the behavior in the docker compose file

Corgam commented:
The branch was renamed.
